networkit/cpp/generators/ErdosRenyiGenerator.cpp

```diff
@@ -22,14 +22,14 @@ Graph ErdosRenyiGenerator::generate() {
 
     {
         ErdosRenyiEnumeratorDefault impl(nNodes, prob, directed);
-        impl.forEdgesParallel([&](int /*tid*/, node u, node v) {
+        impl.forEdges([&](int /*tid*/, node u, node v) {
```

Comment:
This actually makes the ER-Generator slower (because most of the time is spent in generating edges). If it's necessary for the ordering of the resulting adjacency lists, better call `completeGraph` sequentially (with one thread).